webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -81,6 +81,12 @@ var userOverride = {
     };
     $applozic.fn.applozic = function (appOptions, params, callback) {
         var $mck_sidebox = $applozic('#mck-sidebox');
+        if (
+            Object.keys(kommunicate).length !== Object.keys(Kommunicate).length
+        ) {
+            $applozic.extend(Kommunicate, kommunicate);
+            $applozic.extend(kommunicate, Kommunicate);
+        }
```

Comment:
We need to make sure none of the variables are modified after this line is executed. Otherwise there will be a mismatch.